client/src/pages/campaigns/CampaignsContainer.jsx

```diff
@@ -1,21 +1,21 @@
 import React from "react";
 import { useQuery } from "@apollo/react-hooks";
-import { GET_CAMPAIGNS } from "../../gql/queries";
+import { GET_CAMPAIGNS_BY_FILTER } from "../../gql/queries";
 
+import FilterPanel from "./filterPanel";
 import CampaignsTable from "./campaignsTable";
 import CompleteTag from "./campaignsTable/completeTag";
 import Channels from "./campaignsTable/channels";
 import Popover from "./campaignsTable/popover";
 import { numberWithSeparator, convertTimestampToDate } from "../../utils/utils";
 
 const CampaignsContainer = () => {
-  const { data, loading, error } = useQuery(GET_CAMPAIGNS);
+  const { data, loading, error, refetch } = useQuery(GET_CAMPAIGNS_BY_FILTER);
 
-  if (loading) return <p>Loading.. </p>;
   if (error) return <p>ERROR</p>;
   if (!data) return <p>Not found</p>;
 
-  const dataForTable = data.campaigns.map((campaign) => {
+  const dataForTable = data.campaignByFilter && data.campaignByFilter.map((campaign) => {
```

Comment:
to ensure that  data.campaignByFilter has array let's use 'get' method from lodash library.
get(data, 'campaignByFilter', []).map(....)
https://lodash.com/docs/4.17.15#get